cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/coprocessor/ConsumerConfigCache.java

```diff
@@ -105,6 +105,11 @@ public boolean isAlive() {
   }
 
   @Nullable
+  public CConfiguration getCConf() {
+    return (conf != null) ? CConfiguration.copy(conf) : null;
```

Comment:
Since `conf` will be accessed by multiple threads, it is good to make it volatile